install-wings.sh

```diff
@@ -527,6 +587,27 @@ function main {
     if [[ "$CONFIRM_FIREWALL_CMD" =~ [Yy] ]]; then
       CONFIGURE_FIREWALL_CMD=true
     fi
+
+    ask_letsencrypt
+  fi
+
+  if [ "$CONFIGURE_LETSENCRYPT" == true ]; then
+    # set FQDN
+    while [ -z "$FQDN" ] || [ -d "/etc/letsencrypt/live/$FQDN/" ]; do
+        echo -n "* Set the FQDN to use for Let's Encrypt (panel.example.com): "
+        read -r FQDN
+
+        [ -z "$FQDN" ] && print_error "FQDN cannot be empty"
+        [ -d "/etc/letsencrypt/live/$FQDN/" ] && print_error "A certificate with this FQDN already exists!"
```

Comment:
I think this is not the best place to check that. This should be done in `ask_letsencrypt`. Then of course we would need to check if there is something in `/etc/letsencrypt/live/`, not strictly the FQDN that is intended. Then the script would ask: We have found this `$FOUND_FQDN` do you still want to create certificates?

Reply:
I initially tried putting them both in `ask_letsencrypt` but somehow the `if` on line 506 (see [this](https://github.com/vilhelmprytz/pterodactyl-installer/pull/91/files#diff-fd99fe6be85e7380f1f97accfc328d59R507)) in combination with the `while` loop would just make it exit. Is there any real problem having the questions here?

Reply:
Well you get that messages when you are committed to getting certificates. And you would need to exit and restart the script. It would be better to give that information before you press yes to use letsencrypt. Anyway I will also try to get this working.

Reply:
> Well you get that messages when you are committed to getting certificates. And you would need to exit and restart the script. It would be better to give that information before you press yes to use letsencrypt. Anyway I will also try to get this working.

I agree, but I couldn't get it to work properly without strange shortcuts. Since this is my PR you can leave it to me, thanks for the review. I'll make sure I fix it.

Reply:
The problem is that if you put the FQDN/email prompts before asking if enabling Let's Encrypt, then the user has to enter potentially redundant information. A solution would be to first ask whether to enable certificates or not then ask for information and if certificate was found, then re-confirm, which also might look bloated. I will see what I can do.

Reply:
Let me know if I am missing something.